Lombiq.Hosting.Azure.ApplicationInsights/Extensions/ApplicationInsightsInitializerExtensions.cs

```diff
@@ -35,6 +36,24 @@ public static class ApplicationInsightsInitializerExtensions
             .GetSection("OrchardCore:Lombiq_Hosting_Azure_ApplicationInsights");
         applicationInsightsConfigSection.Bind(applicationInsightsOptions);
 
+        if (applicationInsightsOptions.UseEntraAuthentication)
```

Comment:
Use brackets for such larger `if` blocks, even if it's a single expression, for clarity.